Ihor/VanGoux/scss/style.scss

```
@@ -127,111 +278,154 @@ body {
         :nth-child(3) {
             padding-top: 80px;
         }
-        .feat_tab {
-            @include feattab;
+        .feat_card {
+            @include featcard;
+        }
+        .v_divider{
+            display: none;
+        }
+    }
+    @media(max-width: 640px){
+        .features {
+            &_cards {
+                display: flex;
+                flex-wrap: wrap;
+                justify-content: space-between;
+                :nth-child(2){
+                    padding-top: 0;
+                }
+                :nth-child(3){
+                    padding-top: 0;
+                }
+                .feat_card {
+                    @include featcard_mobile;
+                }
+                .v_divider {
+                    height: 60px;
+                    border-right: 2px dotted $blue;
+                    display: inline-block;
+                    margin-left: 60px;
+                }
+            }
+    
         }
     }
 
 }
 
 //vangoux
+
 .vangoux {
     margin-bottom: 100px;
-    &_header {
-        margin-bottom: 50px;
-        h1 {
-        @include h1;
-        margin-bottom: 10px;
-        position: relative;
-        }
-        
-    }
-    .vangoux_divider {
-        @include divider;
-    }
-    &_tabs {
+    &_descr {
         display: flex;
         flex-wrap: wrap;
         justify-content: space-between;
+        :nth-child(3) {
+            margin-right: 0;
+        }
+        :nth-child(6) {
+            margin-right: 0;
+        }
         
         
         
-        .vangoux_tab {
-            max-width: 360px;
+        &_item {
+            width: calc(33% - 20px);
```

Not the best solution. This might work better 
```suggestion
            flex: 1;
            margin-right: 20px;
            &:last-child {
                 margin-right: 0;
            }
```